pkg/prometheus/rules.go

```diff
@@ -16,6 +16,7 @@ package prometheus
 
 import (
 	"fmt"
+	"github.com/prometheus/prometheus/promql/parser"
```

Comment:
This seems like it moved to the wrong place, was this an automated format?

Reply:
It was! fixed now